ui/src/components/mobileservices/MobileServiceView.js

```diff
@@ -25,28 +27,29 @@ class MobileServiceView extends Component {
   }
 
   boundServiceRows() {
-    const rows = [];
-    if (this.props.boundServices) {
-      rows.push(<h2 key="bound-services">Bound Services</h2>);
-      this.props.boundServices.forEach(service => {
-        rows.push(<BoundServiceRow key={service.getId()} service={service} />);
-      });
-    }
-
-    return rows;
+    return (
+      <React.Fragment>
+        <h2 key="bound-services">Bound Services</h2>
+        {this.props.boundServices && this.props.boundServices.length > 0 ? (
+          this.props.boundServices.map(service => <BoundServiceRow key={service.getId()} service={service} />)
+        ) : (
+          <h3>There are no bound services.</h3>
```

Comment:
can we try `<EmptyState>` here and see what's it look like?

Reply:
![image](https://user-images.githubusercontent.com/2912228/49004483-a3af5300-f164-11e8-992d-ddd17c79ead2.png)